packages/plugin-node/src/index.ts

```diff
@@ -407,16 +413,24 @@ export async function build({ workPath }: { workPath: string }) {
       continue;
     }
 
-    await buildEntrypoint({ workPath, entrypoint, config, installedPaths });
+    await buildEntrypoint({
+      vercelConfig,
+      workPath,
+      entrypoint,
+      config,
+      installedPaths,
+    });
   }
 }
 
 export async function buildEntrypoint({
+  vercelConfig,
   workPath,
   entrypoint,
   config,
   installedPaths,
 }: {
+  vercelConfig: any;
```

Comment:
Can we re-use the type from the `client` package?

Reply:
Done in f04411e